pset6/dna/dna.py

```diff
@@ -0,0 +1,46 @@
+from sys import argv, exit
+import csv
+from collections import defaultdict
+
+def main():
+    if len(argv) < 3 or len(argv) > 3:
+        print("Wrong!!!!!")
```

Comment:
Why is this wrong? this edge case should be in the description of the PR or have a comment here

Reply:
Yeah I should upload the actual problem and print a better description to the console. It's wrong because you are supposed to give exactly three commmand line arguments. The name of the python file, the name of the dna text file and the name of the csv file

Reply:
Is there an easy way to change branch names?